code/src/components/TasksDone.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+export const TasksDone = () => {
+  const list = useSelector(store => store.todos.list)
```

Comment:
Since you're only using the items-array, it's better to get only that:
`const items = useSelector(store => store.todos.list.items)`